internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -169,3 +216,34 @@ func parseExpose(line string) []portConfig {
 	}
 	return ports
 }
+
+func parseHealthCheck(line string) (healthCheck, error) {
+	var hc healthCheck
+
+	if line[12:] == "NONE" {
```

Comment:
Why `12` 🤔. What do you think substituting it with a const int with a more readable name? Same with the other integers below. Otherwise you might need to add comments explaining what's happening here.

Reply:
Yep, that sounds good. And the reason is because "Healthcheck " is 12 characters